### ISSUE TITLE
Mac-OS-Setup is not compatible with latest XCode

### ISSUE THREAD

**Issue by lrosenthol** [metanorma/metanorma#274]
I have XCode 14.2 installed, but when running the installer I receive the following error/output.

```
Warning: Your Command Line Tools are too outdated.
Update them from Software Update in System Preferences.

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 13.4.
```

**Comment by ronaldtse** [comment]
Thank you for raising this @lrosenthol -- very honored to have you in our community! 👍 

Just to confirm that you are using the following command to install Metanorma on macOS?
https://www.metanorma.org/install/macos/

```sh
$ brew install metanorma/metanorma/metanorma
```

`Warning: Your Command Line Tools are too outdated.` is a message that commonly appears after a macOS update, or an update of Xcode.

This blog post describes why the warning is raised:
* https://coady.tech/macos-command-line-tools-outdated/

The resolution is typically to run the said commands offered by the message, i.e.:
```
sudo rm -rf /Library/Developer/CommandLineTools 
sudo xcode-select --install 
```

Then:
```
brew update && brew upgrade
```

Then try the install again with:

```sh
$ brew install metanorma/metanorma/metanorma
```

@CAMOBAP can you also help follow up this issue? Thanks!

**Comment by lrosenthol** [comment]
good to be here - thanks!  I had been using my own system for our standards in TC 171 based on Pandoc, but having been spending more time with AsciiDoc (as I used it for the C2PA spec at https://www.c2pa.org), figured it was time to give metanorma a whirl.   and we have some new stuff coming out of SC 29/WG 1, so...

And I see what the problem was.  Even though my XCode app was up to date, I needed to update the command line tools - and your bit did it!   

Thanks!
